contracts/src/Zappers/Interfaces/IZapper.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "./IFlashLoanProvider.sol";
+import "./IExchange.sol";
+
+interface IZapper {
+    struct OpenTroveParams {
+        address owner;
+        uint256 ownerIndex;
+        uint256 collAmount;
+        uint256 boldAmount;
+        uint256 upperHint;
+        uint256 lowerHint;
+        uint256 annualInterestRate;
+        uint256 maxUpfrontFee;
+        address addManager;
+        address removeManager;
+        address receiver;
+    }
+
+    struct CloseTroveParams {
+        uint256 troveId;
+        uint256 flashLoanAmount;
+        uint256 minBoldAmount;
```

Comment:
Is this used for anything? Looks unused to me, but I could be missing something.

Reply:
Good catch, I replaced it by `trove.entireDebt` and forgot to remove this. Thanks!